Catfish/Catfish/Models/Like.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Like.swift
+//  Catfish
+//
+//  Created by morse on 3/31/20.
+//  Copyright © 2020 morse. All rights reserved.
+//
+
+import Foundation
+
+struct Like: Codable {
+    // let comment: Comment
+    // let user: User // Should this be a profile rather than a user?
```

Comment:
This is a great example of why I was thinking of having a `GameUser` or something. This shouldn't be `User` because then we'll need to get the profile for the user to get the right name to show, but it also doesn't feel like it shouldn't be `Profile` simply because it wasn't the profile that liked it...it was the user 🤷‍♂